examples/goconnector/bettors_stream.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/oddin-gg/otsschema/go/oddin.gg/ots"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func connectToRiskApiStream(ctx context.Context, client ots.OtsClient) {
+	closeCh := make(chan bool, 1)
```

Comment:
To signalize simple close request one might use either signaling channel (see below) or even closing of channel (instead of sending any value).

```suggestion
	closeCh := make(chan struct{}, 1)
```

Reply:
done